projects/packages/assets/src/class-assets.php

```diff
@@ -64,18 +64,24 @@ public static function instance() {
 	 * Initalize the hooks as needed.
 	 */
 	private function init_hooks() {
+		global $wp_version;
 		/*
-		 * Load some scripts asynchronously.
+		 * Load some scripts asynchronously. Pre-WordPress 6.3 approach.
 		 */
-		add_filter( 'script_loader_tag', array( $this, 'script_add_async' ), 10, 2 );
+		if( version_compare( $wp_version, '6.3', '<' ) ) {
+			add_filter( 'script_loader_tag', array( $this, 'script_add_async' ), 10, 2 );
+		}
 	}
 
 	/**
 	 * A public method for adding the async script.
 	 *
+	 * @deprecated Since $$next-version$$, the `strategy` feature should be used instead.
```

Comment:
Should we specifically mention to use "defer" with `strategy` to match the old behavior of this function? Since the name is misleading.

Reply:
Would c871128c71b554f7f5de979d071a2a795fa8051c be okay?

Reply:
LGTM